client/src/components/ui/itemrow/RegisterRow.svelte

```diff
@@ -7,23 +8,37 @@
     import { IconSize, RowType, ContextPayload, Events } from '../../types.ts';
     import { Document } from '../../../../../model/src/document.ts';
     import { Category } from '~model/category.ts';
+    import { Snapshot } from '~model/snapshot.ts';
     
     export let iconSize: IconSize;
     export let doc: Document['id'];
     export let category: Category['name'];
     export let title: Document['title'];
-
+    export let creation: Snapshot['creation'];
+    
     const dispatch = createEventDispatcher();
     const rowEvent: ContextPayload = {
         ty: RowType.Inbox,
         id: doc,
     };
+
+    function redirectHandler() {
+        window.location.href = `/track?id=${doc}`;
+    }
+    
+    const docDisplay = `${doc.slice(0,5)}...${doc.slice(-5)}`;
```

Comment:
As mentioned earlier, we should use the CSS [`text-overflow`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow) property instead.